src/app/GitCommands/StringExtensions.cs

```diff
@@ -191,12 +191,12 @@ public static string Quote(this string? s, string q = "\"")
         }
 
         /// <summary>
-        /// Quotes and escapes this string for use as a command line argument.
+        ///  Escapes this string for use as a command line argument.
```

Comment:
nit: Extra space before `Escapes`

Reply:
intended xmldoc style as requested by RussKie

Reply:
It was found through trials and errors that extra space in xml-doc significantly increase readability of the docs.